### PR TITLE
docs: add a content writing guide about documentation/blog/changelog

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

File: content/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+For general writing guide, please check [this writing guide](https://github.com/bytebase/bytebase/blob/main/docs/writing-guide.md).
+
+## General
+
+### Naming
+
+All lowercase with `-` as word separator:
+
+- `my-owesome-post.md`
+- `my-owesome-image.webp`
```
Comment (boojack) [inline content/README.md:18]:
```suggestion
- `my-awesome-post.md`
- `my-awesome-image.webp`
```

**Review by boojack — APPROVED** [review]

File: content/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Blog
+
+- Put under https://github.com/bytebase/bytebase.com/tree/main/content/blog.
+- Under https://github.com/bytebase/bytebase.com/tree/main/static/blog, for each blog post, create a folder using the blog post file name and put all images for that post there.
+- Use feature image.
+
+## Changelog
+
+- Put under https://github.com/bytebase/bytebase.com/tree/main/content/changelog.
+- Under https://github.com/bytebase/bytebase.com/tree/main/static/changelog, for each changlog containing any image, create a folder using the changelog version and put all images for that changelog there.
```
Comment (d-bytebase) [inline content/README.md:42]:
```suggestion
- Under https://github.com/bytebase/bytebase.com/tree/main/static/changelog, for each changelog containing any image, create a folder using the changelog version and put all images for that changelog there.
```

**Review by d-bytebase — APPROVED** [review]

File: content/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+### Naming
+
+All lowercase with `-` as word separator:
+
+- `my-awesome-post.md`
+- `my-awesome-image.webp`
+
+### Image
+
+- Choose readable image name.
```
Comment (milasuperstar) [inline content/README.md:22]:
```suggestion
- Naming: choose a readable image name.
```

File: content/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- Use 16:9 ratio if possible. Feature image **must** use 16:9.
+- Use WebP instead of PNG format to reduce image size.
+- Use GIF judicious. Sometimes you have to use GIF to explain the interaction, however, GIF is much larger and slows page loading speed.
```
Comment (adela-bytebase) [inline content/README.md:25]: Gif could be converted to WebP as well

File: content/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+All lowercase with `-` as word separator:
+
+- `my-awesome-post.md`
+- `my-awesome-image.webp`
+
+### Image
+
+- Choose readable image name.
+- Use 16:9 ratio if possible. Feature image **must** use 16:9.
```
Comment (milasuperstar) [inline content/README.md:23]:
```suggestion
- Sizing: use 16:9 ratio if possible. The feature image sizing **must** be 16:9.
```

File: content/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+- Use WebP instead of PNG format to reduce image size.
```
Comment (milasuperstar) [inline content/README.md:24]:
```suggestion
- Format: use WebP instead of PNG format to reduce image size.
```

**Review by d-bytebase — DISMISSED** [review]

**Review by d-bytebase — APPROVED** [review]

File: content/README.md (hunk 4 above)
Comment (tianzhou) [reply content/README.md]: Split into a separate section

**Comment by tianzhou** [comment]
PTAL

**Review by milasuperstar — APPROVED** [review]

**Review by d-bytebase — APPROVED** [review]

**Review by adela-bytebase — APPROVED** [review]

**Pull request merged**